### DISCUSSION TITLE
Баги с мобами

### DISCUSSION THREAD

**Discussion by MisterN13** [PavelZinchenko/event-horizon-frontier-builds#56] [General]
Уровень 181-2 
![Screenshot_20240325_121308_Event Horizon - Frontier](https://github.com/PavelZinchenko/event-horizon-frontier-builds/assets/161188319/872340ec-068a-4a37-9586-6ccd73639db9)

Бот появляется стреляет био плазменной торпедой по базе и все... Сносит 40 к за 4 секунды и неважно используешь ты бронь или щиты .

Еще
Мобы используют старую железную сферу (этот уровень)

Это так задумано или не?
Видео не грузится

**Comment by vlad-ultramarine** [comment]
Дима попросил посложнее, разраб сделал, хы

**Comment by PavelZinchenko** [comment]
Это не баг, это противники с difficulty class 3, у них открыты все ячейки и улучшения кораблей. Старый iron ball остался в игре, но его больше не купить, а вот враги могут использовать.

Возможно с торпедой перебор получился, но я думаю ее ослабить или сделать защиту против нее.

**Comment by vlad-ultramarine** [comment]
против фиолетового урона есть защита в 50% от самого слабого резиста если их есть 3 вида, интересно как ещё можно от неё защититься? разве что ецм дрон сломает снаряд прежде чем он успеет выпустить облако

**Comment by Inedilarion** [comment]
Торпеде можно сделать ребаланс увеличив радиус поражения, но уменьшив длительность жизни облака. И базам станет легче жить, и против кораблей она станет хоть как-то играть

**Comment by IsmollCheeKen** [comment]
I found the Odin with Heavy Acid Torpedoes very hilarious. It was abused in Arena, and now it's coming to haunt us.

Right now, the only modules that can counter it is the Point Defense System and ECM Drones, however they are not as effective as one may think. Also getting your Star base hit with at least 1 Torpedo is enough to give you a death sentence.

While this certainly presents a problem, I do not agree with reducing the Torpedo's damage or lifetime whatsoever, because this would severely weaken one of the very few viable Null-Type Weapons the game has.

So, I would much prefer it if new and better countermeasures against Heavy Acid Torpedoes are implemented, or maybe increase the acid cloud's velocity so it passes by the Starbase quickly.

**Reply by Inedilarion** [reply]
Increasing the speed of the cloud will reduce its final damage to all enemies due to the reduction in exposure time, you are simply offering the same damage reduction, but in a different way.
And this will just make the weapon even weaker against all enemies, while even now it only works well against bases.
So, in order to make the HAT a truly normal weapon (as you want), it is not enough to nerf its damage, which is why I wrote above about increasing the cloud size to be able to shoot at moving ships more effectively

**Comment by Inedilarion** [comment]
Красные враги есть не у всех фракций и из-за этого в некоторых боях их нет вообще, а в некоторых одновременно может заспавниться штук 5 Ковидов или Одинов. Возможно стоит полностью отвязать их от фракции и спавнить всегда в более-менее одинаковом количестве? Либо добавить такие корабли всем фракциям (может кроме червей, они и сами по себе дают жару)
Кроме того, у меня есть ощущение что на 1 волне спавнится наибольшее количество этих врагов, на 2 меньше и т.д. Посчитать я не смог, так что это не точно. Можно ли сделать фиксированный спавн красных врагов для волн, к примеру, 40%, 55%, 70%, 85%, 100% соответственно?
P.S. Играть стало гораздо веселее, без модулей +20 и с новыми врагами пришлось оптимизировать флот, а против Одинов даже менять базу МК2 на базу с обычной бронёй, раньше такого не бывало)

Not all factions have red enemies, and because of this, in some battles there are none at all, and in some, about 5 Covids or Odins can spawn at the same time. Perhaps it’s worth completely decoupling them from the faction and always spawning them in more or less the same numbers? Or add such ships to all factions (maybe except for worms, they are hot on their own)
In addition, I have a feeling that the largest number of these enemies will spawn on wave 1, 2 less, etc. I couldn't count it, so it's not exact. Is it possible to make a fixed spawn of red enemies for waves, for example, 40%, 55%, 70%, 85%, 100% respectively?
P.S. It became much more fun to play, without +20 modules and with new enemies I had to optimize the fleet, and against the Odins I even changed the MK2 base to a base with regular armor, this has never happened before)

**Reply by MisterN13** [reply]
С красными врагами спавн точно неадекватный.
